src/lib/helpers/ArrowHelper.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+  import { ArrowHelper, Color, ColorRepresentation, Vector3 } from 'three'
+  import Object3DInstance from '../instances/Object3DInstance.svelte'
+
+  //self
+  export let dir:Vector3;
```

Comment:
Please use prettier for formatting and eslint for linting as well as `npm run check` to check for errors or warnings.